### PR TITLE
Issue 4219 - Log internal unindexed searches (notes=A)

### PR DESCRIPTION
**Pull request by vashirov**
Description:
Add a test case.

Relates: https://github.com/389ds/389-ds-base/issues/4219

Reviewed by: ???

### CODE REVIEW

**Review by mreynolds389 — APPROVED** [review]
LGTM

File: dirsrvtests/tests/suites/indexes/regression_test.py [hunk 1]
```diff
@@ -12,14 +12,74 @@
...
+    topo.standalone.restart()
+
+    def fin():
+        """
+        Set nsslapd-idlistscanlimit back to the default value
+        """
+        # If the server crashed, start it again to do the cleanup
+        if not topo.standalone.status():
+            topo.standalone.start()
+        db_cfg.set([('nsslapd-idlistscanlimit', '4000')])
```
Comment (droideck) [inline dirsrvtests/tests/suites/indexes/regression_test.py]: Very minor thing, I think it'll be, probably, better not to hard coded it... 
We can fetch the value at the beginning of the fixture and set it back here at the 'fin()'.

**Review by droideck — APPROVED** [review]
Besides that - looks good to me!

File: dirsrvtests/tests/suites/indexes/regression_test.py [hunk 2]
```diff
@@ -12,14 +12,74 @@
...
+    request.addfinalizer(fin)
+
+
+@pytest.fixture(scope="function")
+def set_small_idlistscanlimit(request, topo):
+    """
+    Set nsslapd-idlistscanlimit to a smaller value to accelerate the reproducer
+    """
+    db_cfg = DatabaseConfig(topo.standalone)
+    db_cfg.set([('nsslapd-idlistscanlimit', '50')])
```
Comment (tmihinto) [inline dirsrvtests/tests/suites/indexes/regression_test.py]: According to the [documentation](https://access.redhat.com/documentation/en-us/red_hat_directory_server/11/html/configuration_command_and_file_reference/database_plug_in_attributes#nsslapd_idlistscanlimit), the min value of 'nsslapd-idlistscanlimit' is 100.

File: dirsrvtests/tests/suites/indexes/regression_test.py (hunk 1 above)
Comment (vashirov) [reply dirsrvtests/tests/suites/indexes/regression_test.py]: Good point, fixed.

File: dirsrvtests/tests/suites/indexes/regression_test.py (hunk 2 above)
Comment (vashirov) [reply dirsrvtests/tests/suites/indexes/regression_test.py]: Nice catch! Fixed.

**Pull request merged**